src/openradardata_validator/odim2ordmsg.py

```diff
@@ -72,7 +76,7 @@ def find_source_type(source: str, sid: str) -> str:
     return ret
 
 
-def odim_datetime(odate: bytes, otime: bytes) -> datetime:
+def odim_datetime(odate: Union[bytes, str], otime: Union[bytes, str]) -> datetime:
```

Comment:
From python 3.10 onwards, this can be written as below. Since python 3.9 is EoL, I think we can use the newer syntax.
```suggestion
def odim_datetime(odate: bytes | str, otime: bytes | str) -> datetime:
```